vips/vip-129/vip-129.ts

```diff
@@ -0,0 +1,129 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+const COMPTROLLER = "0xfd36e2c2a6789db23113685031d7f16329158384";
+const WBETH = "0xa2e3356610840701bdf5611a53974510ae27e2e1";
+const VWBETH = "0x6CFdEc747f37DAf3b87a35a1D9c8AD3063A1A8A0";
+const INITIAL_FUNDING = parseUnits("5.499943", 18);
+const INITIAL_VTOKENS = parseUnits("5.499943", 8);
+const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+const TREASURY = "0xF322942f644A996A617BD29c16bd7d231d9F35E9";
+const RESILIENT_ORACLE = "0x6592b5DE802159F3E74B2486b091D11a8256ab8A";
+const VTOKEN_RECEIVER = "0x7d3217feb6f310f7e7b7c8ee130db59dcad1dd45";
+const BINANCE_ORACLE = "0x594810b741d136f1960141C0d8Fb4a91bE78A820";
+const MAX_STALE_PERIOD = 24 * 60;
```

Comment:
Given the heartbeat for WBET/USD is 15 minutes (https://oracle.binance.com/data-feeds/detail/wbeth-usd). I would set a max stale period of 25 minutes, following what we did for the rest of Binance prices for IL markets [here](https://github.com/VenusProtocol/vips/pull/23/files#diff-0621f3c1304e292463fb1ac783df57984d981c887ca0a1e2a2255bffc3828a01R70)